1-JavaScript/Week2/assignment/ex1-giveCompliment.js

```diff
@@ -18,12 +18,29 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-Java
    Use `console.log` each time to display the return value of the
    `giveCompliment` function to the console.
 -----------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function giveCompliment(name) {
+  const goodWords = [
+    'good',
+    'awSome',
+    'nice',
+    'great',
+    'wellDone',
+    'bad',
+    'real',
+    'cool',
+    'superstar',
+    'super',
+  ];
+
+  const selectRandomly = Math.floor(Math.random() * goodWords.length);
+
+  return `you are ${selectRandomly} , ${name}`;
```

Comment:
Details are important, please read the exercise more carefully.

```suggestion
  return `You are ${selectRandomly}, ${name}!`;
```